cothority.go

```diff
@@ -115,5 +120,8 @@ func runServer(ctx *cli.Context) {
 // signature from each.
 func checkConfig(c *cli.Context) error {
 	tomlFileName := c.String("g")
-	return check.Config(tomlFileName)
+	if c.NArg() > 0 {
+		tomlFileName = c.Args().First()
+	}
+	return check.Config(tomlFileName, c.Bool("detail"))
```

Comment:
yes, it's part of cosi, as it's signing. So it _is_ in a library, but the library is in the cosi-directory.